src/models/predict_model.py

```diff
@@ -170,20 +177,63 @@ def main(args, save_path="", evaluating_test=True):  # noqa: C901
         vv_branch=int(vv_branch),
         De=args.De,
         Do=args.Do,
+        device=device,
     )
 
-    gnn.load_state_dict(torch.load("IN_training/gnn_new_DR0_best.pth"))
-    print(sum(p.numel() for p in gnn.parameters() if p.requires_grad))
-    softmax = torch.nn.Softmax(dim=1)
+    if set_onnx is False:
+        print("11")
+        gnn.load_state_dict(torch.load("../../models/trained_models/gnn_new_best.pth"))
+        print(sum(p.numel() for p in gnn.parameters() if p.requires_grad))
+        #         softmax = torchs.nn.Softmax(dim=1)
+
+        for j in tqdm.tqdm(range(0, target_test.shape[0], batch_size)):
+            dummy_input_1 = torch.from_numpy(test[j : j + batch_size]).to(device)
+            dummy_input_2 = torch.from_numpy(test_sv[j : j + batch_size]).to(device)
+            out_test = gnn(dummy_input_1, dummy_input_2)
+            #             print(np.shape(torch.from_numpy(test[j : j + batch_size])))
+            #             out_test = softmax(gnn(torch.from_numpy(test[j : j + batch_size]).to(device)))
+            out_test = out_test.cpu().data.numpy()
+            #             print(np.shape(out_test))
+            out_test = softmax(out_test, axis=1)
+            if j == 0:
+                prediction = out_test
+            else:
+                prediction = np.concatenate((prediction, out_test), axis=0)
+            del out_test
 
-    for j in tqdm.tqdm(range(0, target_test.shape[0], batch_size)):
-        out_test = softmax(gnn(torch.from_numpy(test[j : j + batch_size]).cuda()))
-        out_test = out_test.cpu().data.numpy()
-        if j == 0:
-            prediction = out_test
-        else:
-            prediction = np.concatenate((prediction, out_test), axis=0)
-        del out_test
+    else:
+        print("22")
```

Comment:
why is this here?

Reply:
I rebased my branch on Ruike's branch. Sorry for the confusion :<